digital-signature/cli/Cargo.toml

```diff
@@ -9,9 +9,8 @@ env_logger = "0.9.0"
 log = "0.4.17"
 digital-signature-methods = { path = "../methods" }
 digital-signature-core = { path = "../core" }
-risc0-zkvm-core = "0.9"
-risc0-zkvm-host = "0.9"
-risc0-zkvm-serde = "0.9"
+risc0-zkvm = "0.11.0-rc.4"
+risc0-zkp = { version = "0.11.0-rc.4", default-features = false}
```

Comment:
Not sure we need to turn off default-features for the host-side code. But not a problem also.